dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpRestClient.internal.cdi-3.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpRestClient.internal.cdi-3.0
+singleton=true
+visibility=private
+-bundles=io.openliberty.org.jboss.resteasy.cdi; apiJar=false; location:="lib/"
```

Comment:
```suggestion
-bundles=io.openliberty.org.jboss.resteasy.cdi
```
apiJar=false and location:="lib" are not needed here.

Reply:
Done